src/fetch-mock.js

```diff
@@ -56,6 +57,16 @@ function mockResponse (url, config) {
 	return Promise.resolve(new Response(body, opts));
 }
 
+function toRequest(url, options) {
+	var request;
+	if (Request.prototype.isPrototypeOf(url) && !options) {
+		request = url;
```

Comment:
Could you change this to not check for non-existence of options, or to throw a descriptive error if url is a request but options exists